.eslintrc

```diff
@@ -0,0 +1,53 @@
+{
+  "ecmaFeatures": {
+    "jsx": true,
+    "modules": true
+  },
+  "env": {
+    "browser": true,
+    "es6": true,
+    "node": true
+  },
+  "parser": "babel-eslint",
+  "rules": {
+    "eol-last": 1,
+    "no-unused-vars": [2, {"varsIgnorePattern": "^React$"}],
+    "max-len": [1, 120, 4],
```

Comment:
If it's not logic, it doesn't belong in a lint file. If you want to be a nazi and dictate the source format, use a beautifier. Also the existing xml formatting in our current setup in blueocean is *all fucked up*